metricbeat/metricbeat.reference.yml

```diff
@@ -280,6 +280,12 @@ metricbeat.modules:
   # Monitoring instead of metricbeat-* indices.
   #xpack.enabled: false
 
+  # Set to cluster to treat each item in the hosts list as an endpoint for
+  # a distinct Elasticsearch cluster, e.g. a load-balancer fronting the cluster.
+  # Set to node (default) to treat each item in the hosts list as an individual
+  # node in the Elasticsearch cluster.
+  #hosts_mode: node
```

Comment:
I wonder if `mode` is enough, in other places we have used `scope` too. Anyway I don't have a strong opinion here

Reply:
I was reserving `mode` for this change: https://github.com/elastic/beats/issues/9424#issuecomment-625375899. In fact, for that other change (#9424).

Using `scope` here instead of `hosts_mode` sounds good to me.

Reply:
Changed in f2a5618.